### PR TITLE
Some intoto refactoring

### PR DESCRIPTION
**Pull request by priyawadhwa**
cc @kommendorkapten I refactored some of the in-toto code to simplify some of the logic, PTAL! I think all of the functionality should (hopefully) be the same, but let me know if something looks off

### CODE REVIEW

File: pkg/chains/formats/intotoite6/intotoite6.go [hunk 1]
```diff
@@ -108,110 +93,80 @@ func (i *InTotoIte6) CreatePayload(obj interface{}) (interface{}, error) {
...
+	att.Predicate.Recipe.DefinedInMaterial = definedInMaterial
+
+	return att, nil
+}
 
-	att.Predicate.Materials = []in_toto.ProvenanceMaterial{}
-	vcsInfo := getVcsInfo(tr)
-	var recipeMaterialUri string
+// materials will add the following to the attestation materials:
+// 	1. All step containers
```
Comment (kommendorkapten) [inline pkg/chains/formats/intotoite6/intotoite6.go:106]: NitPick: extra space

Comment (kommendorkapten) [inline pkg/chains/formats/intotoite6/intotoite6.go:106]:
```suggestion
//  1. All step containers
```

**Comment by kommendorkapten** [comment]
@priyawadhwa Looks good! Just one tiny comment.

File: pkg/chains/formats/intotoite6/intotoite6.go (hunk 1 above)
Comment (priyawadhwa) [reply pkg/chains/formats/intotoite6/intotoite6.go]: thanks!

**Review by dlorenc — APPROVED** [review]

**Pull request merged**